lib/rack/cloudflare_jwt/auth.rb

```diff
@@ -35,21 +33,25 @@ class DecodeTokenError < StandardError; end
       )$
     /x.freeze
 
-    attr_reader :policies
+    attr_reader :policies, :team_domain
 
     # Initializes middleware
     #
     # @example Initialize middleware in Rails
     #   config.middleware.use(
     #     Rack::CloudflareJwt::Auth,
+    #     team_domain:  ENV['RACK_CLOUDFLARE_JWT_TEAM_DOMAIN']
     #     '/admin'   => <cloudflare-aud-1>,
     #     '/manager' => <cloudflare-aud-2>,
     #   )
     #
-    # @param policies [Hash<String, String>] the policies with paths and AUDs.
-    def initialize(app, policies = {})
-      @app      = app
-      @policies = policies
+    # @param [Hash<String, String>] options the options to create a middleware.
+    # @option options [Symbol] :team_domain The Team Domain (e.g. 'test.cloudflareaccess.com')
+    # @option options [String] '<path>' The AUD for current '<path>'.
+    def initialize(app, options = {})
```

Comment:
I thought about:

```suggestion
    def initialize(app, team_domain:, **policies)
```

but ruby 2.6 can't handle it :(

Reply:
or maybe let's use 

```suggestion
    def initialize(app, team_domain, options = {})
```

?

Reply:
I like `initialize(app, team_domain, policies = {})`, since the domain is required 👍

Reply:
I will update.